config/initializers/rack-attack.rb

```diff
@@ -0,0 +1,112 @@
+class Rack::Attack
+  ### Configure Cache ###
+
+  def self.production?
+    Rails.env == 'production' || Rails.env == 'stage'
```

Comment:
`Rails.env.production?`

We never set a server with RAILS_ENV=staging or stage